logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -0,0 +1,19 @@
+package com.wire.kalium.logic.data.message
+
+import com.wire.kalium.logic.data.conversation.ClientId
+import com.wire.kalium.logic.data.conversation.ConversationId
+import com.wire.kalium.logic.data.user.UserId
+
+data class Message(
+    val id: String,
+    val content: MessageContent,
```

Comment:
what if a message has an Image and Text ?

Reply:
Then we can create something like this:

```kotlin

class Image(val imageStuff: ImageStuff, val text: String? = null): MessageContent()

```